fhir/fhir-connector/fhir-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/fhir-operation/component.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+   Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+   WSO2 Inc. licenses this file to you under the Apache License,
+   Version 2.0 (the "License"); you may not use this file except
+   in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied. See the License for the
+   specific language governing permissions and limitations
+   under the License.
+-->
+<component name="fhir-operation" type="synapse/template">
+    <subComponents>
+        <component name="addMetaToResource">
+            <file>addMetaToResource.xml</file>
+            <description>This operation takes a meta, and adds the profiles, tags, and security
+                labels found in it to
```

Comment:
Fixed in https://github.com/wso2-attic/esb-connectors/pull/558/commits/f9e41fea7ca349558494fc0cf418b2fbc71d3511